install/tools/istio_vm_setup.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+#
+# Copyright 2017 Istio Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Script to install istio components for the raw VM.
+
+# Environment variable pointing to the generated Istio configs and binaries.
```

Comment:
directory ? can it really be files ? for example ?

Reply:
For example "." - we'll need to adjust this based on how we distribute, probably a bunch of curl to 
fetch the .deb files, or a tar (as you suggested earlier).
Added a TODO, since I don't know yet.